040-SDK/030-Go/010-overview.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: Xata SDK for Go
+navTitle: SDK overview
+keywords: ['go', 'golang', 'sdk']
+description: Get to know Xata using Go
+slug: sdk/go/overview
+published: true
+---
+
+<Alert status="warning">
+  Please be aware that the Go SDK is still in *alpha*. Not all endpoints are covered in the SDK and potential breaking
+  changes may occur before the promotion to a `1.0.0` and stability GA release.
+</Alert>
+
+The Go SDK is available on GitHub: https://github.com/xataio/xata-go
+
+## Configuration
+
+The client needs a `XATA_API_KEY` environment variables for authentication and the `.xatarc` file for the database URL, please refer to [this page](/docs/getting-started/installation#xatarc) to initialize your project with the `.xatarc` file.
+
+## Example
+
+The following example demonstrates the basic usage of the SDK and how to pull the workspaces and list the first one:
+
+```go
+package main
+
+import (
+  "context"
+  "fmt"
+
+  "github.com/xataio/xata-go/xata"
+)
+
+func main() {
+  workspaces, _ := xata.NewWorkspacesClient()
+  resp, _ := workspaces.List(context.TODO())
+  fmt.Printf("%#v\n", *resp.Workspaces[0])
+}
+```
+
+### Runnable Example
+
+You can run the snippet from above on your machine by, pulling the example code from the xataio/xata-go repository
+
+```bash
+wget -O example.go "https://raw.githubusercontent.com/xataio/xata-go/main/examples/list_workspaces.go"
```

Comment:
Yes, the repo is private at the moment, but once it's public this will work without any further config on your machine necessary. BTW, thank you for testing :)